src/components/molecules/TablesUserList/TablesUserList.tsx

```diff
@@ -80,6 +88,18 @@ const TablesUserList: React.FunctionComponent<PropsType> = ({
   if (!isRecentVenueUsersLoaded) return <>Loading...</>;
 
   const tables: Table[] = customTables || defaultTables;
+
+  const emptyTables = tables.filter((table) => {
+    if (!table.capacity) return false;
+
+    const usersSeatedAtTable = recentVenueUsers.filter(
+      (u) => u.data?.[venueName].table === table.reference
+    );
+    return table.capacity - usersSeatedAtTable.length > 0;
+  });
```

Comment:
This feels.. convoluted.. I wonder if there is a better/more efficient way we can get this?

Reply:
Definitely agree with you! It looks weird. Simplified in aa2e9da5d4c79e5ba7522fb9fefbac09601ebf47